components/expression/expression-panel.html

```diff
@@ -31,7 +43,7 @@
       </tr>
       <tr>
         <td>Condition Labels</td>
-        <td><input type="checkbox" id="label-conditions" title="show experiment condition (condition) labels" checked></td>
```

Comment:
I think it makes more sense to check both genes and condition labels by default.